frontend/javascripts/oxalis/model/sagas/load_histogram_data_saga.js

```diff
@@ -15,17 +15,17 @@ async function fetchAllHistogramsForLayers(
   dataset: APIDataset,
 ): Promise<HistogramDataForAllLayers> {
   const histograms: HistogramDataForAllLayers = {};
-  const histogramPromises = dataLayers.map(async dataLayer => {
+  for (const dataLayer of dataLayers) {
     try {
+      // eslint-disable-next-line no-await-in-loop
```

Comment:
Great find! Please add a comment above the `eslint-disable` line which explains why this is done (basically, what you already included in the PR description), so that it's clear, why this is done sequentially.